test-project/build.gradle.kts

```diff
@@ -0,0 +1,4 @@
+plugins {
+    kotlin("jvm") version "1.5.0"
```

Comment:
I'm aware there is [Compatibility](https://github.com/jeremymailen/kotlinter-gradle#compatibility) section on plugin's Readme file, but as I wasn't sure what would I want to achieve here I decided to use the most recent stable version

Reply:
Also, I named the test app `test-project` and that's a completely separate project that includes the plugin in its buildscript. I don't feel that's the right name, but nothing better came to my mind